### ISSUE TITLE
rettiwt.user.followed() got tweets from people I don't follow

### ISSUE THREAD

**Issue by spencer17x** [Rishikant181/Rettiwt-API#566]
```
const rettiwt = new Rettiwt({
  apiKey: 'xxxx'
});
const result = await rettiwt.user.followed();
```

When I write this, I get tweets from people I don't follow, and each request returns something different.
I couldn't get the right data like I could on the web.

**Comment by spencer17x** [comment]
I found the same code to recommend and unfollowed?
followed: https://github.com/Rishikant181/Rettiwt-Core/blob/dev/src/requests/User.ts#L85-L129
recommended: https://github.com/Rishikant181/Rettiwt-Core/blob/dev/src/requests/User.ts#L381-L426
Params error?

**Comment by spencer17x** [comment]
I think the problem is with the Rettiwt-Core library.
Maybe should use HomeLatestTimeline api.

**Comment by Rishikant181** [comment]
It's an oversight on my part. I made an error while documenting the endpoint in Postman. Working on it now.

The endpoint for followed timeline should have been `.../HomeLatestTimeline` instead of `.../HomeTimeline`.

**Comment by Rishikant181** [comment]
Fixed #567 

@spencer17x Please update to v3.1.1

**Comment by spencer17x** [comment]
> Fixed #567
> 
> @spencer17x Please update to v3.1.1

Nice bro.Thanks.
